src/modules/integrator-guide/pages/corteza-studio/index.adoc

```diff
@@ -0,0 +1,137 @@
+include::ROOT:partial$variables.adoc[]
+
+= Corteza Studio
+
+:description: Corteza Studio allows users to customize Corteza web applications styling according to their preferences.
+Corteza Studio allows users to customize Corteza web applications styling according to their preferences. 
+
+[NOTE]
+====
+For Corteza Studio, to function properly, you need to have dart-sass binaries installed in your system and 
+`HTTP_WEBAPP_ENABLED` enviroment variable set to `true`.
+If you haven't installed it, refer to the <<installing-dart-sass, Installing Dart Sass>> section below.
+====
+
+=== Customizing Corteza Web Applications
+
+To customize your Corteza web application, navigate to menu:User Interface[Settings].
+
+There are two ways of customizing your corteza web applications style:
+
+:sectnums:
+==== Branding
+Under menu:User Interface[Settings] branding section.
+
+image::corteza-studio/branding-section.png[role="data-zoomable"]
+
+A screen showing you all of the available Corteza-specific branding color configuration options.
+
+.From there you can:
+* Click on the rounded color inputs, which will pop up a modal window with a color picker.
+* Here, you can either pick a color from the predefined options or input your own custom color.
+* Finally, click the btn:[Save and close] button to save your input changes.
+
+image::corteza-studio/choosing-color.png[role="data-zoomable"]
+
+After making your branding color changes, click the btn:[Submit] button. This action will save your input changes and automatically refresh your admin web application with the updated branding color.
+
+==== Custom CSS Editor
+Under menu:User Interface[Settings] Custom CSS section.
+
+image::corteza-studio/custom-css.png[role="data-zoomable"]
+
+Custom CSS editor supports both sass and css syntax.
+
+In this section, you can add your Sass or CSS code that specifically targets your web application(s). Afterward, simply click the btn:[Submit] button to save your input changes. This action will save your input changes and automatically refresh your web application with the updated styling.
+
+image::corteza-studio/custom-css-values.png[role="data-zoomable"]
+
+
+====
+Please note that the Custom CSS editor stylesheet code takes precedence over the configuration of branding section variables.
+====
+
+
+== Installing  Dart Sass 
+[[installing-dart-sass]]
+=== Development Enviroment
+To use Corteza, you must install Dart Sass in your development environment. 
```

Comment:
It's no longer required.
Also, here, it's a bit pointless to note all this "that we need it and such"; just focus on how to set it up.
If you wanted to have some _boilerplate padding text_; keep it, else I'd prefer to not see it